hackathon_site/event/views.py

```diff
@@ -203,3 +205,13 @@ class TeamListView(mixins.ListModelMixin, generics.GenericAPIView):
 
     def get(self, request, *args, **kwargs):
         return self.list(request, *args, **kwargs)
+
+
+class TeamIncidentDetailView(mixins.RetrieveModelMixin, generics.GenericAPIView):
```

Comment:
You put this view in the wrong file (nw tho it's confusing) this file is for the jinja views on the hackathon site. Please move this to api_views.py so it's with the other api views.